pkg/controller/storagecluster/reconcile.go

```diff
@@ -33,6 +33,11 @@ type ReconcileStrategy string
 // StorageClassProvisionerType is a string representing StorageClass Provisioner. E.g: aws-ebs
 type StorageClassProvisionerType string
 
+type establisher interface {
```

Comment:
This is a really vague name. What does this do?

Reply:
Right. Basically this interface is responsible for both the create and the delete functions for the resources that ocs-operator has to operate on.

How about CreatorDestroyer as the name? This is the best we could come up with for now, please suggest if you have other options.

Reply:
I'd suggest something like `resourceManager`.

Reply:
Done